packages/consumption/src/modules/requests/local/LocalRequest.ts

```diff
@@ -130,4 +130,18 @@ export class LocalRequest extends CoreSynchronizable implements ILocalRequest {
 
         return false;
     }
+
+    public updateStatusBasedOnTemplateExpiration(templateExpiresAt: CoreDate, comparisonDate: CoreDate = CoreDate.utc()): boolean {
+        if (this.source?.type !== "RelationshipTemplate") return false;
+
+        if (this.status === LocalRequestStatus.Completed || this.status === LocalRequestStatus.Expired) return false;
+        if (this.status === LocalRequestStatus.Decided && this.response?.content.result === ResponseResult.Rejected) return false;
```

Comment:
The idea behind it was, that it is still possible to complete a decided and rejected Request. If the rejection of a Request took place before the RelationshipTemplate was expired and the Request Module is not enabled, I think it should still be possible to complete the Request, even though the RelationshipTemplate is may expired in the mean time. @Milena-Czierlinski What do you think?

(Note: In contrast, for accepted Request this isn't possible, because they can't be completed without a responseSource.)